src/types.jl

```diff
@@ -551,7 +559,7 @@ for (C, acol, cola) in [(DIN99d, :ADIN99d, :DIN99dA),
     cfn = Expr(:tuple, colorfields(C)...)
     elty = eltype_default(C)
     ub   = eltype_ub(C)
-    Csym = C.name.name
+    Csym = isdefined(Core, :UnionAll) ? Base.datatype_name(C) : C.name.name
```

Comment:
sure but it will widen the range of Julia versions on which this package works correctly, which is valuable for bisecting various issues